libqtile/confreader.py

```diff
@@ -49,6 +49,7 @@ class Config:
         "extension_defaults",
         "bring_front_click",
         "wmname",
+        "block_keyboard_on_active_chord",
```

Comment:
Patch itself looks fine, but I wonder if we can make this a property of the particular key chord vs. a global one?

Reply:
I think that this should be True by default globally. The current behavior of passing keys to user applications when a key chord is active seems weird.

We could argue for a use case where a user would still want to send keys to other applications while a chord is active. But that is a niche case and why to pollute the interface for that?

Reply:
> As grabbing keyboard is a bit risky, I don't want to make it default yet. If you think it's fine, I can remove the option to control it.

What risks are you thinking of?

Reply:
This is the way it works today, and we try not to break existing behaviors if possible. Let's just make it a configuration option on KeyChord itself.